cmd/agent/daemon/app/app.go

```diff
@@ -445,8 +427,7 @@ func buildEBPFPolicy(log *logging.Logger, cfg *Config, exporters *state.Exporter
 	}
 
 	dnsEventPolicy := &ebpftracer.EventPolicy{
-		ID:                 events.NetPacketDNSBase,
-		PreFilterGenerator: ebpftracer.DeduplicateDNSEventsPreFilter(log, 100, 60*time.Second),
+		ID: events.NetPacketDNSBase,
```

Comment:
This is a bad idea. Especially with something like `DNS` this will a lot of pressure on the whole event pipeline. The whole idea of the `PreFilterGenerator` was to drop such events as early as possible.

Reply:
I agree, but then the whole events aggregation/deduplication logic should happen inside the tracer. I wanted to keep it simple, but moving it closer would be more efficient.

As for a `lot of pressure` profiles actually shows decrease in usage. I guess moving it tracer would decrease even more.

Reply:
This would explain the dropping of events btw.